apps/server/src/toasts/toasts.controller.ts

```diff
@@ -34,8 +35,19 @@ export class ToastsController {
   }
 
   @Get('happened')
-  async findAllHappened(): Promise<Toast[]> {
-    return await this.toastsService.findAllHappened();
+  async getHappenedToasts(): Promise<Toast[]> {
+    return await this.toastsService.findHappenedToasts();
+  }
+  @Get('current-period-toasts')
+  async countHappenedToastsInLastSixMonths(): Promise<number> {
```

Comment:
I think this name is unnecesarily long. I think you should change it to something like `countToastsInPeriod` or something like that